src/dbg/_global.h

```diff
@@ -2,7 +2,9 @@
 #define _GLOBAL_H
 
 #define _WIN32_WINNT 0x0501
+#ifndef WINVER

```

Comment:
Seems like a toolchain difference. Mine was setting WINVER to 0x601. Since we want to target 501 explicitly, this code is likely the wrong approach. I verified this by turning on 'Show includes' and there were no headers before _global.h in compilation units which displayed the warning.

Not sure why you aren't seeing the redef warning; probably it is just a minor difference in toolchain versions or something. How do you feel about something like this:

```
#ifdef WINVER // Overwrite WINVER if toolchain gives us a default
#undef WINVER
#endif
#define WINVER 0x0501
```

which squashes the warning and applies the right WINVER.